packages/app/source/client/controllers/todos-controller.js

```diff
@@ -6,13 +6,17 @@ Space.Object.extend(Todos, 'TodosController', {
   ],
 
   dependencies: {
-    configuration: 'configuration'
+    configuration: 'configuration',
+    todosStore: 'Todos.TodosStore',
```

Comment:
I think the controller here should just be subscribed to the individual actions of each todo rather than needing to pull in the store again to determine the commands to send